frontend/src/components/QuestionPanel/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import classnames from 'classnames';
+import TextField from '../TextField';
+import './styles.scss';
+
+export enum QuestionPanelColor {
+  Pink = 'pink',
+  Yellow = 'yellow',
+}
+
+export type Question = {
+  id: number;
+  question: string;
+};
+
+type Props = {
+  color: QuestionPanelColor;
+  isEditable: boolean;
+  questions: Question[];
+  onChange?: (value: string, index: number) => void;

```

Comment:
Це, напевно, суб‘єктивна річ, але у меня б був порядок Props згідно їхньою важливістю
```suggestion
  questions: Question[];
  isEditable: boolean;
  color: QuestionPanelColor;
  onChange?: (value: string, index: number) => void;
```

Reply:
Згоден із Андрієм.